specs/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,349 @@
+
+
+CommandBar/CommandBarFlyout updates
+===
+
+# Background
+
+This spec will address three main shortcomings in the
+[CommandBar](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.commandbar?view=winrt-19041)
+and [CommandBarFlyout](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.commandbarflyout?view=winrt-19041)
+space, and propose solutions to these shortcomings. All of these solutions will lead to a more customizable, flexible commanding experience. 
+
+A `CommandBar` provides access to app-level or page-specific commands. 
+A `CommandBarFlyout` is a `CommandBar` that appears in a `Flyout` menu, and must be invoked.
+
+*CommandBar:*
+
+![CommandBar example](images/commandbar-example.png)
+
+*CommandBarFlyout* (invoked for example with a mouse right click):
+
+![CommandBarFlyout example](images/commandbarflyout-example.png)
+
+### Issue #1: You can only use a select few object types inside of a CommandBar/CommandBarFlyout. 
+
+When using the `CommandBar`or `CommandBarFlyout` controls today, the only supported types for items are 
+[AppBarButton](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.appbarbutton?view=winrt-19041), 
+[AppBarToggleButton](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.appbartogglebutton?view=winrt-19041), 
+and [AppBarSeparator](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.appbarseparator?view=winrt-19041). 
+When you use an AppBarButton in a `CommandBar` or `CommandBarFlyout`, all of the interactions and styles needed to conform with
+the parent control are included - height, width, the background color when hovered/pressed, the font size, etc. 
```

Comment:
Took me a while to figure out what "included" meant.

```suggestion
the parent control are provided automatically - height, width, the background color when hovered/pressed, the font size, etc. 
```

Reply:
Recommend